dags/bqetl_main_summary.py

```diff
@@ -430,6 +430,21 @@
         task_concurrency=1,
     )
 
+    telemetry_derived__hcm_clients__v1 = bigquery_etl_query(
+        task_id="telemetry_derived__hcm_clients__v1",
+        destination_table="hcm_clients_v1",
+        dataset_id="telemetry_derived",
+        project_id="moz-fx-data-shared-prod",
+        owner="dthorn@mozilla.com",
+        email=[
+            "dthorn@mozilla.com",
+            "jklukas@mozilla.com",
```

Comment:
```suggestion
```

Reply:
will do, but it will apply to the entire dag